Project organization/Project structure.md

```diff
@@ -49,7 +49,7 @@ It is a good practice to keep app packages updated, and for installing and updat
 
 #### Test projects
 
-Regardless of the test framework we are using, we try to keep a simple test project structure. For each app project, we create additional test projects. We use simple naming strategies for the projects, only adding the suffix ".Test" for the test project. For example:
+Regardless of the test framework, we try to keep a simple test project structure. For each app project, we create additional test projects. We use simple naming strategies for the projects, only adding the suffix ".Test" for the test project. For example:
 
 Example.API would be Example.API.Test
```

Comment:
Since you're here, I don't see any reason why should this example be in a separate row, so maybe we can fix the formatting here.